include/swift/AST/diagnostics/en.yaml

```diff
@@ -4292,6 +4292,10 @@
   msg: >-
     use unwrapped type %0 as key path root
 
+- id: optional_keypath_application_base 
+  msg: >-
+    use '?' to access key path subscript only for non-'nil' base values
```

Comment:
`en.yaml` has been moved to `localization/diagnostics/` so you'd have to rebase :/

Reply:
Ah sure, no problem :)